screens/ProfileSetup.tsx

```
@@ -0,0 +1,369 @@
+
+    const goToNotifications = () => {
+        router.replace('/(account)/notifications');
+    };
+
+    return (
+        <ScrollView style={styles.container}>
+            <Text style={styles.title}>Welcome to Vested!</Text>
+
+            {/* Step 1 */}
+            <View style={styles.section}>
+                <Text style={styles.sectionTitle}>Step 1: Tell us why you're here</Text>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)